WordPress/src/main/AndroidManifest.xml

```diff
@@ -477,6 +467,19 @@
                 android:host="wordpress.com"
                 android:pathPattern="/notifications/.*"
                 android:scheme="http" />
+
+                <data
+                    android:host="wordpress.com"
+                    android:pathPattern="/media/..*"
+                    android:scheme="http" />
+
```

Comment:
Hey @zwarm 
When I was testing the changes, `wordpress.com/media` deeplink was not working for me. I added a path pattern of  `android:pathPattern="/media/.*"` to fix that. Commit 05189a62eeb62cfa8a634a77697060c342628330. Hope thats ok.

Reply:
That's weird ... we don't support .* for pages or stats or others.